Signal/src/views/OWSSystemMessageCell.m

```diff
@@ -0,0 +1,205 @@
+//
+//  Copyright (c) 2017 Open Whisper Systems. All rights reserved.
+//
+
+#import "OWSSystemMessageCell.h"
+#import "NSBundle+JSQMessages.h"
+#import "TSUnreadIndicatorInteraction.h"
+#import "UIColor+OWS.h"
+#import "UIFont+OWS.h"
+#import "UIView+OWS.h"
+#import <JSQMessagesViewController/UIView+JSQMessages.h>
+#import <SignalServiceKit/TSCall.h>
+#import <SignalServiceKit/TSErrorMessage.h>
+#import <SignalServiceKit/TSInfoMessage.h>
+
+@interface OWSSystemMessageCell ()
+
+@property (nonatomic) UIImageView *imageView;
+@property (nonatomic) UILabel *titleLabel;
+
+@end
+
+#pragma mark -
+
+@implementation OWSSystemMessageCell
+
++ (NSString *)cellReuseIdentifier
+{
+    return NSStringFromClass([self class]);
+}
+
+- (void)configure
+{
+    self.backgroundColor = [UIColor whiteColor];
+
+    if (!self.titleLabel) {
+        self.imageView = [UIImageView new];
+        [self.contentView addSubview:self.imageView];
+
+        self.titleLabel = [UILabel new];
+        self.titleLabel.textColor = [UIColor colorWithRGBHex:0x403e3b];
+        self.titleLabel.font = [OWSSystemMessageCell titleFont];
+        self.titleLabel.numberOfLines = 0;
+        self.titleLabel.lineBreakMode = NSLineBreakByWordWrapping;
+        [self.contentView addSubview:self.titleLabel];
+    }
+
```

Comment:
nit: Do we want to `OWSAssert(self.interaction?)` or we could make this method `configureWithInteraction:`